### PR TITLE
Onboarding jitNetworkAccessPolicies to a new stable API version - 2020-01-01

### PR DESCRIPTION
**Pull request by avamitay**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by avamitay** [comment]
Can someone please add the 'DoNotMerge' label? I do not have permissions.

**Comment by avamitay** [comment]
@akning-ms Can you please also add a 'DoNotMerge' label?

File: specification/security/resource-manager/Microsoft.Security/stable/2020-01-01/jitNetworkAccessPolicies.json [hunk 1]
```diff
@@ -0,0 +1,770 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Security Center",
+    "description": "API spec for Microsoft.Security (Azure Security Center) resource provider",
+    "version": "2015-06-01-preview"
```
Comment (chlahav) [inline specification/security/resource-manager/Microsoft.Security/stable/2020-01-01/jitNetworkAccessPolicies.json]:
```suggestion
    "version": "2020-01-01"
```

**Comment by akning-ms** [comment]
> No commit pushedDate could be found for PR 8719 in repo Azure/azure-rest-api-specs

@NullMDR  Can you help take a look why this message?

**Comment by NullMDR** [comment]
@akning-ms The head branch is deleted so we get that failure.

**Review by KrisBash — APPROVED** [review]
Looks to be the same as previously reviewed APIs

**Comment by akning-ms** [comment]
Hi @avamitay, what is current status of this PR?  Do you want merge? Looks it is pending for quite long.

**Comment by avamitay** [comment]
@akning-ms this PR has been abandoned, I apologize for not closing it earlier. Thanks!

**Pull request closed without merging**